.github/workflows/pull_request.yml

```diff
@@ -6,6 +6,22 @@ on:
       - master
 
 jobs:
+  test_2x:
```

Comment:
@macumber: reinstated v291 tests (fine with gh-pages branch solution, something of an issue with master/docs solution as jekyll requires Ruby 2.3)